Dockerfile

```diff
@@ -7,6 +7,7 @@ COPY build/shared /app/build/shared
 COPY node_modules /app/node_modules
 COPY bin/_run-docker /app/bin/
 COPY build/screenshots.xpi /app/build/xpi/screenshots.xpi
+ENV NODE_ICU_DATA="/app/node_modules/full-icu"
```

Comment:
@chenba I'm a little confused by the comments in bug #2216. If the problem was a node version mismatch, do you still need to add this env var?

Reply:
Yes.  Still need this for node.  The path was correct but the version of ICU data was incorrect last time around because CircleCI was using a different version of node than Docker.  (The data is pulled down during a postinstall script into `node_modules/full-icu`; it's not dependent on the full-icu module version.)  If I've left that env var along with the wrong version of ICU data, node won't run at all, so it was removed.